Rockerfile

```diff
@@ -0,0 +1,14 @@
+# build image
+FROM golang:latest
+RUN go get -v github.com/mailhog/MailHog && \
+    cd /go/src && \
+    CGO_ENABLED=0 go build -a -installsuffix cgo -v -o /go/bin/MailHog.o github.com/mailhog/MailHog/main.go
```

Comment:
use `go get` to download the main.go and all dependencies first.
then build the binary.

Since alpine linux and therefor gliderlabs/alpine docker containers use musl instead of gnu libc your golang binaries will not work inside alpine. so we need build with static linking enabled.
see also: http://www.blang.io/posts/2015-04_golang-alpine-build-golang-binaries-for-alpine-linux/